package.json

```diff
@@ -37,6 +37,7 @@
 		"@descope/audit-management-widget": "0.1.0",
 		"@descope/role-management-widget": "0.1.35",
 		"@descope/user-management-widget": "0.4.36",
+		"@descope/user-profile-widget": "0.0.15",
```

Comment:
Update all other widgets and web-component

Reply:
done